app/controllers/uphold_controller.rb

```diff
@@ -0,0 +1,57 @@
+class UpholdController < ApplicationController
```

Comment:
I would prefer to not have two controllers named the same, can we rename one of these `UpholdConnectionsController`?